Tests/Fixtures/TestProject/AnotherProject/AnotherProject.xcodeproj/project.pbxproj

```diff
@@ -41,6 +41,23 @@
 		};
 /* End PBXGroup section */
 
+/* Begin PBXLegacyTarget section */
+		A6D9FB94860C005F0B723B5F /* IncludedLegacy */ = {
+			isa = PBXLegacyTarget;
+			buildConfigurationList = AC68886F4CEE08D3593D0877 /* Build configuration list for PBXLegacyTarget "IncludedLegacy" */;
+			buildPhases = (
+				69078D1DA3F942D5BC752081 /* Sources */,
+			);
+			buildToolPath = /usr/bin/true;
+			buildWorkingDirectory = .;
```

Comment:
Is this supposed to be `AnotherProject/.`, or is this correct?

Reply:
This is correct — when generating AnotherProject directly from the AnotherProject folder, the working directory for the target is ‘.’, but when including it in another project in a different directory, *that* project’s target will have a working directory relative to the other project.

In other words, regardless of where the generated project ends up, the working directory for the target declared in this folder will evaluate to this folder